card/mod/standard/set/all/path.rb

```diff
@@ -29,6 +29,10 @@
   # @option opts [Symbol] :format
   # @option opts [Hash] :card
   # @option opts [TrueClass] :no_mark
+
+  CAST_PARAMS = { slot: { hide: :array, show: :array, wrap: :array } }.freeze
```

Comment:
seems at bit overengineered to me. To what else can you cast a param in a url?

Reply:
I thought the casting was overkill in the first place :).

It seemed plausible to me that we might have some string representation for a hash.  And we could use something similar to force `item=bar` into `items[view]=bar`.  But meh.

Reply:
My main aversion was to hard-coding slot handling in the path handling, so I was mostly trying to start down a path of configurability.